### PR TITLE
[Clojure] init config.json and implementing-a-concept-exercise.md

### PR DESCRIPTION
**Pull request by bemself**
The implementing-a-concept-exercise.md is not fully done yet, would like to init it though, and have more experienced team member to do together~

### CODE REVIEW

File: languages/clojure/config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "language": "Clojure",
+  "active": true,
+  "blurb": "a JVM hosted functional programming language.",
+  "version": 1,
```
Comment (ErikSchierboom) [inline languages/clojure/config.json]:
```suggestion
  "version": 3,
```

File: languages/clojure/docs/implementing-a-concept-exercise.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+## Step 1: adding track-specific files
+
+These files are specific to the Clojure track:
+
+- `<NAME>.clj`: the Clojure file.
+- `<NAME>_test.clj`: the test suite.
+
+## Step 2: adding common files
+
+How to create the files common to all tracks is described in the [how to implement a concept exercise document][how-to-implement-a-concept-exercise].
```
Comment (ErikSchierboom) [inline languages/clojure/docs/implementing-a-concept-exercise.md]: The [how-to-implement-a-concept-exercise] link is missing (see bottom of https://raw.githubusercontent.com/exercism/v3/master/languages/csharp/docs/implementing-a-concept-exercise.md).

File: languages/clojure/config.json (hunk 1 above)
Comment (bemself) [reply languages/clojure/config.json]: @ErikSchierboom thank you for pointing this out, actually i'm not sure what this version is for, is it for clojure or clojure in exercism? If it is the former, I see clojure version is now at 1.10.*, please correct me if wrong:-)

Comment (NobbZ) [reply languages/clojure/config.json]: It is the version of the configuration format, which is co-incidentally equal to the version of exercism its targetting.

Comment (bemself) [reply languages/clojure/config.json]: @NobbZ got it, corrected, thank u~

File: languages/clojure/docs/implementing-a-concept-exercise.md (hunk 2 above)
Comment (bemself) [reply languages/clojure/docs/implementing-a-concept-exercise.md]: @EduardoBautista corrected, thank u~

**Comment by ErikSchierboom** [comment]
@bemself Are there any active Clojure maintainers apart from yourself?

**Comment by bemself** [comment]
@ErikSchierboom Yes I think @porkostomus is also a V3 maintainer for clojure, who is much more experienced than I~

**Comment by ErikSchierboom** [comment]
@bemself Alright. Let's wait for @porkostomus to review then.

**Pull request merged**